src/main/java/ErrorHandling/UnknownCommand.java

```diff
@@ -0,0 +1,7 @@
+package ErrorHandling;
+
+public class UnknownCommand extends Exception{
+    public UnknownCommand(){
+        // No code for now
```

Comment:
This class does not include new properties or behaviors apart from its parent class. It could be much better to [leave this out](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#delete-dead-code).